devtools/cli/src/test/java/io/quarkus/cli/CliProjectGradleTest.java

```diff
@@ -54,16 +55,36 @@ void startGradleDaemon(boolean useWrapper) throws Exception {
             gradle = ExecuteUtil.findExecutableFile("gradle");
         }
 
-        CliDriver.Result result = CliDriver.executeArbitraryCommand(gradle.getAbsolutePath(), "--daemon", "-q",
-                "--project-dir=" + project.toAbsolutePath());
+        List<String> args = new ArrayList<>();
+        args.add(gradle.getAbsolutePath());
+        args.add("--daemon");
```

Comment:
Stupid question maybe but why do we start a daemon?

Reply:
I don't know, maybe @ebullient knows more? but I think we could get rid of it.

Reply:
It is almost impossible to NOT start the daemon. So what ends up happening (when you don't start the daemon) is that it starts/stops one for every single test (which is slower than just managing the daemon). It is really really (really!) annoying, and there are a lot of complaints about it, but seemingly no interest in _actually_ being daemon free with Gradle.